docs/docker.md

```diff
@@ -8,12 +8,11 @@ on Docker Hub at [timescale/promscale](https://hub.docker.com/r/timescale/promsc
 To fetch the latest Promscale image, please refer to the [releases](https://github.com/timescale/promscale/releases)
 section of Promscale github repository.
 
-A docker image of TimescaleDB with the `promscale`
-extension is available at on Docker Hub at
-[`timescaledev/promscale-extension:latest-ts2-pg13`](https://hub.docker.com/r/timescaledev/promscale-extension). You can
-install this via `docker pull timescaledev/promscale-extension:latest-ts2-pg13`.
+A docker image of TimescaleDB with associated extensions including the `promscale`
+extension is available at on Docker Hub at [`timescale/timescaledb-ha:pg14-latest`](https://hub.docker.com/r/timescale/timescaledb-ha).
+You can install this via `docker pull timescale/timescaledb-ha:pg14-latest`.
 
-The `ts2-pg13` suffix means TimescaleDB version 2 on PostgreSQL version 13. We also publish images to DockerHub for other combinations of TimescaleDB and PostgreSQL versions in case you have specific version requirements.
+The `pg14` suffix means this image is for PostgreSQL version 13. We also publish images to DockerHub for other PostgreSQL versions in case you have specific version requirements.
```

Comment:
pg14 is for PG version 13?
```suggestion
The `pg14` suffix means this image is for PostgreSQL version 14. We also publish images to DockerHub for other PostgreSQL versions in case you have specific version requirements.
```